### PR TITLE
Update 7.10.2 changelog

### PR DESCRIPTION
**Pull request by nkammah**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@
 
 * 7.10.2 as default version.
 
+
+| PR | Author | Title |
+| --- | --- | --- |
+| [#753](https://github.com/elastic/ansible-elasticsearch/pull/753) | [@jmlrt](https://github.com/jmlrt) | Fix java install path when system has multiple java  |
```
Comment (jmlrt) [inline CHANGELOG.md:10]: Do you know why https://github.com/elastic/ansible-elasticsearch/commit/470eae90ede5af4fbe9c501707d9d68b9abb0115 is missing here?

**Review by jmlrt — CHANGES_REQUESTED** [review]

**Review by jmlrt — APPROVED** [review]
LGTM⛴

**Comment by jsermer** [comment]
It appears 7.10.2 now predates 7.10.1 (meaning the number of commits to master is greater where it should be less or 0).  Even changing to v7.10.2 branch shows latest commit was 10/22/2020 whereas v7.10.1 shows latest commit as 12/9/2020

**Comment by jmlrt** [comment]
@jsermer, yes it is, we had somme issue during 7.10.2 release for this Ansible role and should fix it pretty soon.

**Comment by jmlrt** [comment]
@jsermer, this should be fixed now, you can remove the role and download it again.

**Comment by jsermer** [comment]
Thanks for the quick turnaround

**Pull request merged**